### ISSUE TITLE
Migrate the Discover and manage applications with New Relic content to the new developer site

### ISSUE THREAD

**Issue by jpvajda** [newrelic/developer-website#466]
### Summary

1. Migrate the [Discover and manage applications](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/build-new-relic-one/discover-manage-applications-new-relic-one-catalog) with New Relic content to the new developer site and locate this content under the [Build Apps section](https://developer.newrelic.com/build-apps) as a new guide called 

**Manage your applications**

_with the following sections_

**Add images and metadata to your apps**

**Handling duplicate apps**

**Subscribing and unsubscribing apps**

2. Then rework the Docs page to point to the developer site. 

With that "Build on New Relic One" section, remove "Nerdpack file structure" page and redirect to the dev site.
we should have one page on building on building apps in that section that basically says, you can learn all about it at the developer site and points the developer site.

### Motivation

This will keep all the Build Apps content on the developer site and remove a redundant page on the Docs site.

**Comment by mmfred** [comment]
I will do this with work in this ticket: https://newrelic.atlassian.net/browse/DOC-5926

**Comment by mmfred** [comment]
@jpvajda - I did this a little differently than described. I added the info about subscribing and unsubscribing to the current page, Serve, publish, and deploy apps (https://developer.newrelic.com/build-apps/publish-deploy). For the permissions stuff, I created a new page, https://developer.newrelic.com/build-apps/permission-manage-apps. I put this second page in the build-apps area, but it might better belong in the explore-docs area.
